.circleci/config.yml

```diff
@@ -33,14 +33,33 @@ jobs:
             - node_modules
           key: v1-dependencies-{{ checksum "package-lock.json" }}-{{ checksum "combined-package-lock.txt" }}
 
-      - run:
-          name: Run tests with coverage
-          command: pushd api && npm run test-coverage && popd
-
       - persist_to_workspace:
           root: ~/repo
           paths: .
 
+  test:
+    machine: # Docker is available in CircleCI's machine executor
+      image: ubuntu-1604:201903-01
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v12.13 && nvm alias default v12.13
+            npm i && pushd api && npm install && popd && pushd authorizer && npm install && popd
```

Comment:
what does `popd` mean?

Reply:
pushd - is like cd with a history (i.e. it pushes a directory on to the stack)
popd - changes your directory to where you were previously (i.e. it pops it off the stack)

Useful commands to know 👍